frontend/web/components/modals/CreateGroup.js

```diff
@@ -162,11 +169,15 @@ const CreateGroup = class extends Component {
                     }}
                     value={name}
                     onChange={(e) =>
-                      this.setState({ name: Utils.safeParseEventValue(e) })
+                      this.setState({
+                        emailEdited: true,
```

Comment:
This should be `nameEdited` or `groupNameEdited`, right? We're not editing any emails here unless I'm mistaken?